### DISCUSSION TITLE
openssl command line removed in 1.4

### DISCUSSION THREAD

**Discussion by sguedon** [nginx-proxy/nginx-proxy#2371] [Q&A] [answered]
Hi Team,

Any reason to remove openssl command line (at least in alpine version) ?

**Comment by buchdag** [comment]
Hi. I believe it was previously installed only because it was thought to be a dependency of nginx on Alpine (it isn't).

**Reply by sguedon** [reply]
Any plan to restore it ? usefull to create self signed certificates ;-)

**Reply by buchdag** [reply]
I'll check the size difference between the image with and without the openssl CLI, if it's small enough I'll revert the change.

**Reply by sguedon** [reply]
Great, let me know 👍

**Reply by buchdag** [reply]
That's a .8 MB difference (1.3 % increase) on the image size, I think we can live with that.

**Reply by buchdag** [reply, marked as answer]
Openssl CLI added back in #2372
